opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -126,11 +98,30 @@ const extractTokenFromBasicAuth = async (authorization) => {
 
 export const findById = async (context, user, userId) => {
   const data = await storeLoadById(context, user, userId, ENTITY_TYPE_USER);
+  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && user.id !== userId) {
```

Comment:
does it mean that an orga admin can administer itself ?

Reply:
Since we use findById to get the logged user (and get its capas, organizations, administrated_organizations through buildCompleteUser), this condition was added to not check that the fetched user has common orgas with the logged user if it's himself.
In the line 103, user.administrated_organizations was empty and we ended up with ForbiddenAccess each time

Reply:
so the admin orga can administer itself ? I mean he can edit his own user, since userEdit is accessible with orga admin capability.

Reply:
I added some security checks (in the API) and now, an organization admin can administrate itself but won't be able to change group or organization